extensions/machine-learning-services/src/serverConfig/serverConfigManager.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
```

Comment:
You don't need these - they're always added by the compiler anyways for our projects as long as you're using the shared config (which you are) [https://github.com/microsoft/azuredatastudio/blob/master/extensions/shared.tsconfig.json#L6](https://github.com/microsoft/azuredatastudio/blob/master/extensions/shared.tsconfig.json#L6)